client/flutter/lib/pages/main_pages/app_tab_router.dart

```diff
@@ -40,6 +41,52 @@ class AppTabRouter extends StatelessWidget {
 
   AppTabRouter(this.tabs, this.navItems);
 
+  @override
+  _AppTabRouterState createState() => _AppTabRouterState(tabs, navItems);
```

Comment:
Instead of passing params, use `this.widget.tabs` and `this.widget.navItems` in the state class. See [this answer](https://stackoverflow.com/a/50818870/2434476) for detail

```suggestion
  _AppTabRouterState createState() => _AppTabRouterState();
```